admin/user

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash -eu
+
+#
+# User manager.
+#
+
```

Comment:
This script repeats a functionality of xyz.openbmc_project.User.Manager service.
I think it would be better to use D-Bus interfaces instead direct calls of the system utilities.

Reply:
Yes, I have thought about that too.
But the service has complex interface, it is hard to communicate with them via bash. The second thing - it doesn't cover all our requirements.
I would suggest using this script now, later we can create a standalone module to manage user accounts via console.

Reply:
I fear that the user created by system tools will not be managed by service until it restarted.

Reply:
It's true, D-Bus service doesn't know about new accounts until restart.

Reply:
But it is not a problem - we can restart the service from the script.

Reply:
Done. Works as expected with service restart.